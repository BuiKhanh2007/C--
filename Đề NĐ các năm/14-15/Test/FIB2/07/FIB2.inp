10
7 15
26 127577
15 340
15 456
19 3019
20 6599
15 539
8 18
32 736586
28 59218
