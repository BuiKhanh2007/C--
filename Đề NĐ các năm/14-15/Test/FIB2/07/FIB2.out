6
48730
130
174
1153
2521
206
7
281351
22619
